### PR TITLE
Update styles

### PR DESCRIPTION
**Pull request by phelian23**
Pull request for the desktop version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -53,6 +55,7 @@ ul {
   font-weight: normal;
   color: #fff;
   font-size: 20px;
+  width: 566px;
 }
```
Comment (SajjadAhmad14) [inline styles.css:60]: Please align the mailbox icon with other nav items vertically. Kindly see the screenshot below for reference.
menu

File: styles.css [hunk 2]
```diff
@@ -100,7 +107,7 @@ nav {
 .workhead {
```
Comment (SajjadAhmad14) [inline styles.css:112]: Please use font from the design for work heading

File: styles.css [hunk 3]
```diff
@@ -304,7 +311,7 @@ button:disabled {
   display: inline-block;
 }
```
Comment (SajjadAhmad14) [inline styles.css:321]: Please use the font for the paragraph of the `About me` section from the design page.

File: styles.css [hunk 4]
```diff
@@ -447,7 +452,7 @@ button:disabled {
   font-weight: normal;
   font-size: 15px;
   color: #979493;
-  width: 600px;
+  width: 588px;
   padding: 14px;
 }
```
Comment (SajjadAhmad14) [inline styles.css:466]: Please use more padding between icons. Kindly see the screenshot below for reference
icons
.

File: styles.css [hunk 5]
```diff
@@ -39,6 +39,8 @@ ul {
   font-weight: normal;
```
Comment (SajjadAhmad14) [inline styles.css:39]: Please add unnecessary files and folders like `vscode` to `gitignore`.

File: styles.css [hunk 6]
```diff
@@ -39,6 +39,8 @@ ul {
...
   color: #f55800;
```
Comment (SajjadAhmad14) [inline styles.css:40]: Please add instructions on how to run the project locally in README.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Femi Awoyemi , 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AshakaE — APPROVED** [review]
### Hi @phelian23  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL]
   - You can deploy your project with github pages 


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**